lib/blueprinter/helpers/base_helpers.rb

```diff
@@ -62,6 +63,44 @@ def validate_root_and_meta(root, meta)
         end
       end
 
+      def dynamic_blueprint?(blueprint)
+        blueprint.is_a?(Proc)
+      end
+
+      def validate_blueprint(blueprint, method)
```

Comment:
Would you please add the bang to each of these validation methods that possibly throw errors
```suggestion
      def validate_blueprint!(blueprint, method)
```

Reply:
Yes. I had initially not added the bang to the end of the method names because I noticed that [`validate_root_and_meta`](https://github.com/procore/blueprinter/blob/eb46aef170b29a3df29d741fc249713382cff0c4/lib/blueprinter/helpers/base_helpers.rb#L54) didn't have it.  Should be good now.